src/ripple/protocol/impl/tokens.cpp

```diff
@@ -77,13 +97,17 @@ digest2 (Args const&... args)
     @note This checksum algorithm is part of the client API
 */
 void
```

Comment:
The `checksum` functions should probably be marked `static`

Reply:
Addressed in 4ecdd58c54c23ffbb297be83b8252e53631287fe